src/main/java/duke/storage/Storage.java

```diff
@@ -0,0 +1,175 @@
+package duke.storage;
+
+import duke.exception.DukeException;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+import duke.task.TaskList;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.FileReader;
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Storage {
+    String filePath;
+
+    /**
+     * Create a storage object with given file path.
+     *
+     * @param  filePath   the file path to saved task list
+     */
+    public Storage(String filePath) {
+        this.filePath = filePath;
+    }
+
+    /**
+     * Get the string format of a list of task objects.
+     *
+     * @param  tasks   the list of task objects
+     * @return  listString    the list of tasks in string format
+     */
+    public String convertListToString(TaskList tasks) {
+        String tasksString = "";
+        for (int i = 0; i < tasks.size(); i++) {
+            Task task = tasks.get(i);
+            int isDone = (task.getIsDone()) ? 1 : 0;
+            if (task instanceof Deadline) {
+                tasksString += "D | "
+                        + isDone
+                        + " | "
+                        + task.getDescription()
+                        + " | "
+                        + ((Deadline) task).getDate()
+                        + " "
+                        + ((Deadline) task).getTime();
+            } else if (task instanceof Event) {
+                tasksString += "E | "
+                        + isDone
+                        + " | "
+                        + task.getDescription()
+                        + " | "
+                        + ((Event) task).getDate()
+                        + " "
+                        + ((Event) task).getFromTimeToTime();
+            } else if (task instanceof Todo) {
+                tasksString += "T | "
+                        + isDone
+                        + " | "
+                        + task.getDescription();
+            }
```

Comment:
it would be better to have a method in every sub class of task to convert the task to string of the saving format.